packages/0x.js/src/index.ts

```diff
@@ -6,8 +6,6 @@ export {
     ECSignature,
     ZeroExError,
     EventCallback,
-    EventCallbackAsync,
-    EventCallbackSync,
```

Comment:
Can you also open a PR in the website repo, removing these from the `public0xjsTypes` array in `utils/constants.ts`.

Reply:
No. We shouldn't remove them cause they're still present in the old versions